pstats_print2list/pstats_print2list.py

```diff
@@ -5,7 +5,16 @@
 import os
 import pstats
 import re
-import StringIO
+try:
+    import StringIO
```

Comment:
Is not from StringIo import StringIO to have the same effect?

Reply:
Done